products/workers/src/content/platform/limits.md

```diff
@@ -294,15 +288,9 @@ Workers KV is an eventually consistent system, meaning that reads will sometimes
 
 ## Durable Objects
 
-<Aside type="warning" header="Beta">
-
-Durable Objects are currently in beta. See [their docs](/learning/using-durable-objects) for more information about the beta and any current limitations.
-
-</Aside>
-
 - Unlimited Durable Objects within an account or of a given class
 
-- 10 GB total storage per account (this will be raised after the beta period)
+- 10 GB total storage per account
```

Comment:
So we're sticking with 10GB for now? Dropping the beta comment makes this feel more permanent/unchangeable than it was before. Should we note that people can ask for it to be raised?

Reply:
Oh, just saw "if you need more storage, please contact us." below, I think it should be mentioned here too.

Reply:
Preferably with some sort of way to contact us so that folks don't give up before they find a way to do so.